pkg/driver/node/mounter/systemd_mounter.go

```diff
@@ -49,7 +49,7 @@ func (m *SystemdMounter) IsMountPoint(target string) (bool, error) {
 		return false, err
 	}
 
-	mountPoints, err := m.MountLister.ListMounts()
+	mountPoints, err := m.Mounter.List()
```

Comment:
How does this work? I did a quick check here https://github.com/kubernetes/mount-utils/blob/887f9a11be38008c0ba049117324b7f645ab5ea2/mount_linux.go#L426-L427 and it looks like it's just parsing `/proc/mounts`?

Reply:
Thanks for reviewing. So our current code was basically copy-paste of `k8s.io/mount-utils` because we were mounting `/proc/mounts` for our container from host at `/host/proc/mounts` and that library has [hardcoded path ](https://github.com/kubernetes/mount-utils/blob/887f9a11be38008c0ba049117324b7f645ab5ea2/mount_linux.go#L46) for `/proc/mounts`. With this change, we no longer mount `/host/proc/mounts` but just enable propagation from host to our `/proc/mounts`, so we can just use the library as is without any modifications. Which is achieved by [Mount Propagation](https://kubernetes.io/docs/concepts/storage/volumes/#mount-propagation) --updates our containers' `/proc/mounts` from host whenever there is a new mount is the on `/var/lib/kubelet`.